test/runtests.jl

```diff
@@ -32,7 +33,7 @@ using Test
 
     @testset "organise_obs" begin
 
-        c_organise_obs(args...; kwargs...) = collect(organise_obs(args...; kwargs...))
+        corganise_obs(args...; kwargs...) = collect(organise_obs(args...; kwargs...))
```

Comment:
Why this change? It seemed more readable before

Reply:
Good catch it was a bad find and replace on my part